### ISSUE TITLE
Problem accessing PhoneGap APIs from remote JS

### ISSUE THREAD

**Issue by cronoklee** [TruckMovers/cordova-plugin-remote-injection#29]
I'm trying to access PhoneGap plugins from javascript on my remote site but I cant seem to get window.plugins to be anything but undefined. Here's my setup in config.xml:

```
<content src="https://domain.com/path-to-app " />
<plugin name="cordova-plugin-remote-injection" />
<preference name="CRIPageLoadPromptInterval" value="5" />
<preference name="CRIShowConnectionErrorDialog" value="0" />
```

I'm getting  `Uncaught SyntaxError: Invalid or unexpected token` in console at the end of the first line:
`script.src="data:text/javascript;charset=utf-8;base64,Ly8gUG...`

There seems to be a fix for this here: https://github.com/TruckMovers/cordova-plugin-remote-injection/pull/25. @bradleyjames Can you merge this fix into the main build or is there some other workaround?

I presume that the **CRIInjectFirstFiles** preference is optional and not required in my case? It's not really clear from the docs what this param is intended for or if it's required.

**Comment by bradleyjames** [comment]
Have you verified that #25 fixes your issue?  If so I'll merge it in.  I don't have android setup at this point for testing and time to spend on getting it setup is nonexistent.

All preference are optional.  I'll update the README to include this information.

**Comment by cronoklee** [comment]
Thanks for the reply Bradley. I dont have the setup here for custom plugins so I can't verify 100% that this will fix my issue, but I am getting the same "unexpected token" error at the very end of the base64 line so it is almost definitely the same issue. Could you maybe merge it and I'll revert immediately with an update?

**Comment by bradleyjames** [comment]
Sorry for dragging my feet here.  I'll get this merged and released soon.

**Comment by bradleyjames** [comment]
Merged and released as v0.5.2. Sorry this took so long.

**Comment by cronoklee** [comment]
Thanks a lot Brad!
